files/default/kibana-4.x-init

```diff
@@ -1,157 +0,0 @@
-#!/bin/sh
```

Comment:
same with this file. the kibana .deb package installs it's own service init script.